src/main/java/com/synopsys/integration/jenkins/scan/global/ApplicationConstants.java

```diff
@@ -37,5 +25,31 @@ public class ApplicationConstants {
     public static final String NOT_AVAILABLE = "NA";
     public static final String ENV_JOB_NAME_KEY = "JOB_NAME";
     public static final String ENV_CHANGE_ID_KEY = "CHANGE_ID";
+    public static final String ENV_BRANCH_NAME_KEY = "BRANCH_NAME";
+
+    public static final String SCAN_TYPE_KEY = "scan_type";
+    public static final String BLACKDUCK_URL_KEY = "blackduck_url";
+    public static final String BLACKDUCK_API_TOKEN_KEY = "blackduck_api_token";
+    public static final String BLACKDUCK_INSTALL_DIRECTORY_KEY = "blackduck_install_directory";
+    public static final String BLACKDUCK_SCAN_FULL_KEY = "blackduck_scan_full";
+    public static final String BLACKDUCK_SCAN_FAILURE_SEVERITIES_KEY = "blackduck_scan_failure_severities";
+    public static final String BLACKDUCK_AUTOMATION_FIXPR_KEY = "blackduck_automation_fixpr";
+    public static final String BLACKDUCK_AUTOMATION_PRCOMMENT_KEY = "blackduck_automation_prcomment";
+
+    public static final String COVERITY_CONNECT_URL_KEY = "coverity_connect_url";
+    public static final String COVERITY_CONNECT_USER_NAME_KEY = "coverity_connect_user_name";
+    public static final String COVERITY_CONNECT_USER_PASSWORD_KEY = "coverity_connect_user_password";
+    public static final String COVERITY_CONNECT_PROJECT_NAME_KEY = "coverity_connect_project_name";
+    public static final String COVERITY_CONNECT_STREAM_NAME_KEY = "coverity_connect_stream_name";
+    public static final String COVERITY_CONNECT_POLICY_VIEW_KEY = "coverity_connect_policy_view";
+    public static final String COVERITY_INSTALL_DIRECTORY_KEY = "coverity_install_directory";
+    public static final String COVERITY_AUTOMATION_PRCOMMENT_KEY = "coverity_automation_prcomment";
+
+    public static final String BITBUCKET_TOKEN_KEY = "bitbucket_token";
+
+    public static final String BRIDGE_DOWNLOAD_URL = "bridge_download_url";
```

Comment:
Please rename to: 
synopsys_bridge_download_url, 
synopsys_bridge_download_version, 
synopsys_bridge_install_directory ( for synopsys_bridge_path)
bridge_include_diagnostics

All blackduck, coverity, polaris parameters should be appended with bridge_

Reply:
As discussed, this will be refactored with a different PR which will be associated with another ticket.